test/fork/migration-v2.js

```diff
@@ -616,36 +595,7 @@ describe('V2 upgrade', function () {
     await tx.wait();
   });
 
-  // TODO review
-  it.skip('unlock claim assessment stakes', async function () {
-    const stakesPath = path.join(config.paths.root, 'scripts/v2-migration/output/eligibleForCLAUnlock.json');
-    const claimAssessors = require(stakesPath).map(x => x.member);
-
-    const tcNxmBalance = await this.nxm.balanceOf(this.tokenController.address);
-
-    console.log('Token balances before running tc.withdrawClaimAssessmentTokens');
-    console.log({
-      tcNxmBalance: tcNxmBalance.toString(),
-    });
-
-    const totalToProcess = claimAssessors.length;
-    console.log(`Processing withdrawClaimAssessmentTokens for ${totalToProcess} claim assesors`);
-    let amountProcessed = 0;
-    while (claimAssessors.length > 0) {
-      const batchSize = 100;
-      const batch = claimAssessors.splice(0, batchSize);
-      await this.tokenController.withdrawClaimAssessmentTokens(batch);
```

Comment:
Are we disabling this to avoid gas costs or there are other reasons?

Reply:
I don't see good reasons to do it. Any manual action is prone to errors, and we need to maintain the data structures and logic around locked tokens anyway due to the CN deposits that are stored similarly. I'm trying to minimize error surface first and foremost.

Reply:
FYI I just found out there were 2 duplicates in the list of generated addresses...